src/LabLog/Migrations/20170928082432_InitialCreate.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+using System.Collections.Generic;
+
+namespace LabLog.Migrations
+{
+    public partial class InitialCreate : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "LabEvents",
+                columns: table => new
+                {
+                    Version = table.Column<int>(type: "INTEGER", nullable: false)
+                        .Annotation("Sqlite:Autoincrement", true),
+                    EventBody = table.Column<string>(type: "TEXT", nullable: true),
+                    EventType = table.Column<string>(type: "TEXT", nullable: true),
```

Comment:
Not nullable for event type

Reply:
Automatically generated?

Reply:
Maybe [Required] or [Required()] above the property in the class file? I've seen conflicting info.
Can't get it to work either way though. Getting
`'Required' is not an attribute class`
Tried a bunch of using statements at the top based on info from web, but dunno